docs/snippets/ov_extensions.py

```diff
@@ -38,3 +38,32 @@ def conversion(node):
 
 core.add_extension(ConversionExtension("ThresholdedRelu", conversion))
 #! [py_frontend_extension_ThresholdedReLU]
+
+
+#! [py_frontend_extension_aten_hardtanh]
+import torch
+from openvino.frontend import ConversionExtension, NodeContext
+from openvino.tools.mo import convert_model
+
+
+class HardTanh(torch.nn.Module):
+    def __init__(self, min_val, max_val):
+        super(HardTanh, self).__init__()
+        self.min_val = min_val
+        self.max_val = max_val
+
+    def forward(self, inp):
+        return torch.nn.functional.hardtanh(inp, self.min_val, self.max_val)
+
+
+def convert_hardtanh(node: NodeContext):
+    inp = node.get_input(0)
+    min_value = node.get_values_from_const_input(1)
+    max_value = node.get_values_from_const_input(2)
+    return ops.clamp(inp, min_value, max_value).outputs()
```

Comment:
Is `.outputs()` really required? Can the node object be converted automatically to a vector of outputs?

Reply:
Yes, it's required.